package.json

```diff
@@ -1,67 +1,72 @@
 {
-  "name": "@passport-next/passport",
-  "version": "2.1.1",
-  "description": "Simple, unobtrusive authentication for Node.js.",
-  "keywords": [
-    "express",
-    "connect",
-    "auth",
-    "authn",
-    "authentication"
-  ],
-  "author": {
-    "name": "Rowan Wookey",
-    "email": "admin@rwky.net"
-  },
-  "contributors": [
-    "Brett Zamir"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/passport-next/passport.git"
-  },
-  "bugs": {
-    "url": "http://github.com/passport-next/passport/issues"
-  },
-  "homepage": "https://github.com/passport-next/passport",
-  "license": "MIT",
-  "main": "./lib",
-  "browserslist": [
-    "node >= 8.0.0"
-  ],
-  "dependencies": {
-    "@passport-next/passport-strategy": "1.x.x"
-  },
-  "devDependencies": {
-    "@mysticatea/eslint-plugin": "^10.0.3",
-    "@passport-next/chai-passport-strategy": "1.x.x",
-    "chai": "4.x.x",
-    "chai-connect-middleware": "0.x.x",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb-base": "13.x.x",
```

Comment:
This config is no longer needed unless you'd like to add it--probably as the first item in the `extends` array within `.eslintrc.js` to allow the other configs to override it where stricter.

In such a case, you could also get rid of the rules following the comment `// add back different or stricter rules from airbnb`. I added them there explicitly, so you could see what the stricter or different rules from airbnb were from ash-nazg and if you preferred ash-nazg's approach, just remove them (and drop the airbnb devDep. entirely).